libs/dal/src/repositories/change/change.schema.ts

```diff
@@ -0,0 +1,41 @@
+import * as mongoose from 'mongoose';
+import { Schema, Document } from 'mongoose';
+import { schemaOptions } from '../schema-default.options';
+import { ChangeEntity } from './change.entity';
+
+const changeSchema = new Schema(
+  {
+    enabled: {
+      type: Schema.Types.Boolean,
+      default: false,
+    },
+    type: Schema.Types.String,
+    createdAt: {
+      type: Schema.Types.Date,
+      default: Date.now,
+    },
```

Comment:
```suggestion
    createdAt: {
      type: Schema.Types.Date,
      default: Date.now,
    },
```

We don't really need this, since { ... schemaOptions } handles this directly